packages/world/src/modules/registration/RegistrationModule.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { RegistrationSystem } from "./RegistrationSystem.sol";
+
+import { Call } from "../../Call.sol";
+import { ROOT_NAMESPACE } from "../../constants.sol";
+import { WithMsgSender } from "../../WithMsgSender.sol";
+import { Resource } from "../../types.sol";
+
+import { IModule } from "../../interfaces/IModule.sol";
+
+import { ResourceType } from "./tables/ResourceType.sol";
+import { SystemRegistry } from "./tables/SystemRegistry.sol";
+
+/**
+ * The RegistrationModule installs the RegistrationSystem
+ * and all required tables and function selectors in the World.
+
+ * Note:
+ * This module is required to be delegatecalled via `World.registerRootSystem`.
+ * because otherwise the table libraries would try to call methods on the World contract
+ * that are only registered in this module (e.g. `registerTable`).
+ * If the module is delegatecalled, the StoreCore functions are used directly.
+ */
+contract RegistrationModule is IModule, WithMsgSender {
+  error RegistrationModule_OnlyRoot();
```

Comment:
this doesn't seem used? and also isn't very self explanatory on its own

Reply:
good catch! This was used in some iteration step but isn't needed anymore